vendor/github.com/OpenBazaar/multiwallet/client/blockbook/client.go

```diff
@@ -551,6 +562,12 @@ func (i *BlockBookClient) setupListeners() error {
 	i.listenLock.Lock()
 	defer i.listenLock.Unlock()
 
+	// Add stored watch addresses to listenQueue if there are any
+	i.listenQueue = append(i.listenQueue, i.listenAddrs...)
```

Comment:
Let's move this down to where the `listenQueue` is processed so we can see more obviously that the `listenAddrs` are appended before processing.

Reply:
Fixed